docs/src/basics/exporters.md

```diff
@@ -0,0 +1,89 @@
+# Exporters
+
+As Zeebe processes jobs and workflows, or performs internal maintenance (e.g.
+raft failover), it will generate an ordered stream of records:
+
+![record-stream](/basics/exporters-stream.png)
+
+While the clients provide no way to inspect this stream directly, Zeebe can load
+and configure user code that can process each and every one of those records, in the form of an exporter.
+
+An **exporter** provides a single entry point to process every record that is
+written on a stream.
+
+With it, you can:
+
+* Persist historical data by pushing it to an external data warehouse
+* Export records to a visualization tool (e.g. [zeebe-simple-monitor](https://github.com/zeebe-io/zeebe-simple-monitor/))
+
+You can find a reference exporter implementation in the form of the Zeebe-maintained
+[ElasticSearch exporter](https://github.com/zeebe-io/zeebe/tree/{{commit}}/exporters/elasticsearch-exporter).
+
+For more information, you can read the [reference information page](reference/exporters.html).
+
+> **Note**: it is not recommended to use exporters for orchestration purposes, as
+> that is what workflows are for.
+
+## Usage
+
+Zeebe will only load exporters which are configured through the main Zeebe TOML
+configuration file.
+
+Exporters are configured as TOML maps, with a standard preamble describing the
+exporter:
+
+```toml
+[[exporters]]
+id = "myExporter"
+className = "com.myCompany.MyExporter"
+```
+
+Optionally, an exporter can be loaded through an external JAR file, simply by
+pointing to it through the configuration:
+
+```toml
+[[exporters]]
+id = "myExporter"
+className = "com.myCompany.MyExporter"
+jarPath = "exporters/com.myCompany.exporters.jar"
+```
+
+Additionally, you can configure exporter-specific arguments through a nested map
+under the label `[exporters.args]`:
+
+```toml
+[[exporters]]
+id = "myExporter"
+className = "com.myCompany.MyExporter"
+jarPath = "exporters/com.myCompany.exporters.jar"
+  [exporters.args]
+  username = "mmustermann"
+  host = "udp://myCompany.com:8080"
+```
+
+You should refer to an exporter's documentation on how to configure it.
+
+Once an exporter is configured, the next time Zeebe is started, the exporter
+will start receiving records. Note that it is only guaranteed to see records
+produced from that point on.
+
+## Considerations
+
+The main impact exporters have on a Zeebe cluster is that they remove the burden
+of persisting data indefinitely.
+
+Once data is not needed by Zeebe itself anymore, it will query its exporters to
+know if it can be safely deleted, and if so, will permanently erase it, thereby
+reducing disk usage.
+
+Therefore, if no exporters are configured at all, then Zeebe will automatically
+erase data when it is not necessary anymore. If you need historical data, an exporter is thus necessary.
```

Comment:
This is a key point for this Basics section for me. `No exporter => no history` is an important concept.

Reply:
Keep in basics but emphasize (blockquote)